.github/workflows/cfn-publish-resource.yaml

```diff
@@ -10,6 +10,10 @@ on:
         description: "the folder name of the Resource in this Repo."
         default: ""
         required: true
+      resourceVersionPublishing:
+        description: "the version used to publish the resource, empty will use the next minor version."
+        default: ""
+        required: false
```

Comment:
One question is not related to these changes but related to the action. I would expect the action to install Atlas CLI as well since it is used for the CFN test. Where is the AtlasCLI installed?

Reply:
in the SSM doc: https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/cfn-publishing-automation/cfn-3P-provider-register.yaml#L208
using the script: https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/cfn-install-mongo-cli.sh

Reply:
Oh I see, another improvement would the remove that script and use https://github.com/mongodb/atlas-github-action in the action instead similar to AWS CLI

Reply:
the publish is run from SSM doc using Codebuild